### PR TITLE
Add credits page

### PR DESCRIPTION
**Pull request by alshakerM**

### CODE REVIEW

File: components/Credits/Credits.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { GitHub, Twitter } from '@mui/icons-material';
+import styles from './Credits.module.css';
+
+export function Credits() {
+  return (
+    <div className={styles.contentContainer} id="credits">
+      <h1 className={styles.title}>Mente</h1>
+      <div className={styles.iconsContainer}>
+        <a href="https://github.com/alshakerM" target="_blank" rel="noreferrer">
```
Comment (alshakero) [inline components/Credits/Credits.js]: This should lead to the repo of the project, not your profile.

**Review by alshakero — CHANGES_REQUESTED** [review]
This is perfect but it's just a footer. All you need to do is rename it to "Footer". 

It's my bad that I didn't explain that "Credits" is a boring page that I haven't designed yet. It will contain the credits for the photos and meditation lessons, some lessons require attribution, and we should show their license.

File: components/Credits/Credits.js (hunk 1 above)
Comment (alshakerM) [reply components/Credits/Credits.js]: On it

**Review by alshakero — CHANGES_REQUESTED** [review]
This is still called Credits. It should be Footer.

File: components/Footer/Footer.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { GitHub, Twitter } from '@mui/icons-material';
+import styles from './Footer.module.css';
+
+export function Footer() {
+  return (
+    <div className={styles.contentContainer} id="credits">
```
Comment (alshakero) [inline components/Footer/Footer.js]:
```suggestion
    <div className={styles.contentContainer}>
```

**Review by alshakero — APPROVED** [review]

**Pull request merged**